app/screens/home/DetailsScreen.tsx

```diff
@@ -379,6 +382,32 @@ console.log(videoIsFocused,"..videoIsFocused");
     trEvenBackground: 'transparent',
     // fontFamily: '"Open Sans"' // beware to quote font family name!
   });
+
+  // method use for highlight text
+  const highlightWord = (content: string, query: string): any => {
+    if (typeof content !== 'string') {
+      return "";
+    }
+
+    const regex = new RegExp(`${query}`, 'gi');
+    const highlightedContent = content.replace(regex, '<span style="background-color: rgba(255, 141, 107, 0.4);">$&</span>');
+
+    return highlightedContent;
+  }
+  // method use for highlight text without image
+  const highlightTextWithoutImages = (jsonContent: any, wordToHighlight: any): any => {
```

Comment:
Method name is self explaining. We can remove the comment or rephrase it. 
Let's use string for parameters and return type.